.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,59 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, master ]
```

Comment:
master?

Reply:
Oops this was autogenerated by github... will fix